src/changelog.rs

```diff
@@ -0,0 +1,38 @@
+use near_sdk::borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::NearSchema;
+
+use crate::*;
+
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, NearSchema)]
+#[serde(crate = "near_sdk::serde")]
+#[borsh(crate = "near_sdk::borsh")]
+pub struct ChangeLog {
+    pub block_id: u64,
+    pub block_timestamp: u64,
+    pub changed_object_id: u32,
```

Comment:
What is the point of using `changed_object_id` if it is included inside the change_log_type? I suggest to drop it.